app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt

```diff
@@ -67,199 +72,264 @@ fun UserProfilePreview() {
-                      text = "Followers",
-                      modifier = Modifier.align(Alignment.CenterHorizontally),
-                      style = AppTypography.categoryTextStyle)
-                }
-            Column(
-                modifier = Modifier.align(Alignment.CenterVertically).padding(horizontal = 30.dp)) {
-                  Text(
-                      text = "NBR",
-                      modifier = Modifier.align(Alignment.CenterHorizontally),
-                      style = AppTypography.bigNumberStyle)
-                  Text(
-                      text = "Following",
-                      modifier = Modifier.align(Alignment.CenterHorizontally),
-                      style = AppTypography.categoryTextStyle)
+
+
+            Scaffold(
+                topBar = {},
+                bottomBar = { NavigationBar(navigation) },
+                modifier = Modifier.fillMaxSize()
+            ) { innerPadding ->
+                Column(modifier = Modifier.padding(innerPadding)) {
+                    Row(modifier = Modifier.height(75.dp).fillMaxSize()) {}
+
+                    // Profile picture and name (later maybe more informations depending of data classes
+                    // updates
+                    Row(modifier = Modifier.align(Alignment.CenterHorizontally)) {
+                        // Profile picture
+                        Column() {
+                            AsyncImage(
+                                model = profile.profileImageUrl,
+                                contentDescription = "Profile picture",
+                                placeholder = painterResource(id = R.drawable.blankprofile),
+                                modifier =
+                                Modifier.shadow(
+                                    elevation = 15.dp,
+                                    shape = CircleShape,
+                                    ambientColor = Color.Black,
+                                    spotColor = Color.Black
+                                )
+                                    .size(110.dp)
+                                    .clip(CircleShape),
+                                contentScale = ContentScale.Crop
+                            )
+                        }
+                        // Other informations
+                        Column() {
+                            Text(
+                                text = profile.username, // I think we only show the pseudo here and keep birthdate
+                                // name and surname private.
+                                style =
+                                TextStyle(
+                                    fontSize = sizeUsername,
+                                    lineHeight = 16.sp,
+                                    fontFamily = FontFamily(Font(R.font.montserrat)),
+                                    fontWeight = FontWeight(700),
+                                    color = Color.Black,

```

Comment:
The color should be generic so that it changes with the dark mode (md_theme_light_dark)